package.json

```
@@ -15,7 +15,7 @@
     "dev": "npx ergol application/app.ts -c ./config/ergol.json",
     "test": "npm run test:lint && npm run test:unit && npm run test:functional",
     "test:unit": "mocha",
-    "test:functional": "cucumber-js --exit --fail-fast",
+    "test:functional": "cucumber-js --require-module ts-node/register --require features/step_definitions/*.ts --exit --fail-fast",
```

Nitpicking: Shorthand
```suggestion
    "test:functional": "cucumber-js -r ts-node/register --require features/step_definitions/*.ts --exit --fail-fast",
```